src/components/tx-flow/flows/TokenTransfer/CreateTokenTransfer.tsx

```diff
@@ -129,34 +126,75 @@ const CreateTokenTransfer = ({
             )}
           </FormControl>
 
-          <Controller
-            name={TokenTransferFields.tokenAddress}
-            control={control}
-            rules={{ required: true }}
-            render={({ fieldState, field }) => (
-              <FormControl fullWidth>
-                <InputLabel id="asset-label" required>
-                  Select an asset
-                </InputLabel>
-                <Select
-                  labelId="asset-label"
-                  label={fieldState.error?.message || 'Select an asset'}
-                  error={!!fieldState.error}
-                  {...field}
-                  onChange={(e) => {
-                    field.onChange(e)
+          <FormControl
+            className={css.outline}
+            fullWidth
+            sx={{
+              mt: 3,
+              borderColor: isAmountError ? 'error.main' : 'border.main',
```

Comment:
This element even has a class already, please move the CSS there.

For the border color, as well as the InputLabel text color, you could toggle a class on the FormControl instead of individual CSS props.